content/tests/components/pages/OverviewPage.test.tsx

```diff
@@ -35,10 +41,67 @@ describe("OverviewPage", () => {
       .spyOn(NormandyAPI.prototype, "fetchRecipe")
       .mockImplementation(() => Promise.resolve(recipe));
 
-    const { getByText } = renderWithContext(<OverviewPage />);
-    await waitForElementToBeRemoved(getByText(/Loading Overview/));
+    jest
+      .spyOn(ExperimenterAPI.prototype, "fetchExperiments")
+      .mockImplementation(() => Promise.resolve([]));
+
+    const doc = renderWithContext(<OverviewPage />);
+    await waitForElementToBeRemoved(doc.getByText(/Loading Overview/));
 
     expect(NormandyAPI.prototype.fetchRecipe).toBeCalledTimes(1);
-    expect(getByText(recipe.id.toString())).toBeInTheDocument();
+    expect(doc.getByText(recipe.id.toString())).toBeInTheDocument();
+  });
+
+  it("should display overdue recipes", async () => {
+    const recipe1 = recipeFactory.build({
+      latest_revision: { enabled: true },
+    });
+
+    const recipe2 = recipeFactory.build({ latest_revision: { enabled: true } });
+
+    const recipe3 = recipeFactory.build({
+      latest_revision: { enabled: false },
+    });
+
+    const exp1 = experimenterResponseFactory.build({ normandy_id: recipe1.id });
+    const exp2 = experimenterResponseFactory.build({ normandy_id: recipe2.id });
+    const exp3 = experimenterResponseFactory.build({ normandy_id: recipe3.id });
```

Comment:
This works fine, you may be interested in the `buildMany` variant of the deafult factory though.

```tsx
const recipes = recipeFactory.buildMany([
	{ latest_revision: { enabled: true } },
	{ latest_revision: { enabled: true } },
	{ latest_revision: { enabled: false } },
]);
const experiments = experimentResponseFactory.buildMany(recipes.map((r) => ({ normandy_id: r.id })));
```